src/components/ArticleList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Box, Grommet } from "grommet";
+import { grommet } from "grommet/themes";
+import axios from "axios";
+
+class ArticleList extends Component {
+  state = {
+    articleList: []
+  };
+
+  componentDidMount() {
+    axios.get("/articles").then(response => {
+      this.setState({
+        articleList: response.data.articles
+      });
+    });
+  }
+  render() {
+    let articleDisplay;
+    if (this.state.articleList !== []) {
+      articleDisplay = this.state.articleList.map(article => {
+        return (
+          <>
+            <Grommet full theme={grommet}>
```

Comment:
Move this down to the Return instead. That way it won't create 2 separate container divs. Boxes are enough to space them out